frontend/test/metabase/scenarios/admin/settings/settings.cy.spec.js

```diff
@@ -198,7 +198,9 @@ describe("scenarios > admin > settings", () => {
 
     cy.findByText("January 7, 2018").click({ force: true });
     cy.findByText("2018/1/7").click({ force: true });
-    cy.contains("17:24 (24-hour clock)").click();
+    cy.wait("@saveFormatting");
```

Comment:
@ariya this was the needed fix. Cypress was too fast and Metabase didn't have enough time to update date formatting. Hence the failure from the screenshot.

Reply:
Nice catch @nemanjaglumac !!